packages/cli/src/commands/create/index.ts

```diff
@@ -46,6 +49,16 @@ export async function createTutorial(flags: yargs.Arguments) {
     return 0;
   }
 
+  applyAliases(flags);
+
+  try {
+    verifyFlags(flags);
```

Comment:
It's not possible to do `--start --no-install`. You can't start the project without installing the dependencies. If that happens, we throw. This method could have additional checks in the future.